errors/errors.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package errors
+
+import "fmt"
+
+// Error specifies an API that must be fullfiled by error type
+type Error interface {
+
+	// Error implements the error interface.
+	Error() string
+
+	// Msg returns error message
+	Msg() string
+
+	// Err returns wrapped error
+	Err() *customError
```

Comment:
Should `Err()` be exported or we're only using it in `Contains`? If we're using it only in `Contains`, we don't need it at all.

Reply:
Keeping it